### ISSUE TITLE
Alt text is dropped when a video with custom thumbnail is uploaded.

### ISSUE THREAD

**Issue by ScreenDream** [mastodon/mastodon#31680]
### Steps to reproduce the problem

1. Drag and drop a video in the new post area to upload it.
2. Click on the edit pencil icon of the video to edit the Alt text.
3. Enter Alt Text.
4. Upload a custom thumbnail for the video and save the changes.
5. If you now post your video post, the Alt text is missing.
...

### Expected behaviour

The Alt text should of course be kept alive and kicking ;-)

### Actual behaviour

The Alt text is dropped.

### Detailed description

When using a custom thumbnail for videos, the Alt text vanishes when the toot is posted.

This only happens when adding a custom thumbnail.
I can work around it if after posting I re-edit the post, enter the Alt text again and re-post.
I have tested this on mastodon.art, zirk.us and Vivaldi.social.
The bug is around for as long as I can remember, several years I think.

### Mastodon instance

mastodon.art, zirk.us and Vivaldi.social

### Mastodon version

v4.2.12 and earlier

### Browser name and version

Vivaldi for the last year or more, Firefox

### Operating system

Windows 10 and 11

**Comment by ClearlyClaire** [comment]
I haven't identified it yet, but I can confirm I can reproduce the issue. The whole modal is being reworked, though, so I may wait for that to be done before trying to fix it.
